devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -93,16 +99,17 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
         BipedalLocomotion::YarpUtilities::VectorsCollectionClient client;
         BipedalLocomotion::YarpUtilities::VectorsCollectionMetadata metadata;
         std::string signalName;
+        unsigned int numMissedPackets;
```

Comment:
As explained below I would remove this information

Reply:
Better to discuss face-to-face